CollAction/Controllers/ProjectsController.cs

```diff
@@ -412,13 +413,31 @@ public ViewResult StartInfo()
 
             if (success)
             {
-                string confirmationEmail = 
-                    "Hi!<br><br>" +
+                string projectUrl = this.Url.Action("Details", "Projects", new { name=commitProjectViewModel.ProjectName}, HttpContext.Request.Scheme); 
```

Comment:
Bit nitpicky, probably more consistent to add a couple of spaces around = and }:

``new { name = commitProjectViewModel.ProjectName }``

Reply:
fixed